documentation-support/src/test/java/tech/picnic/errorprone/documentation/Compilation.java

```diff
@@ -48,7 +48,8 @@ public static void compileWithDocumentationGenerator(
             "-Xlint:all,-serial",
             "-Xplugin:DocumentationGenerator -XoutputDirectory=" + outputDirectory,
             "-XDdev",
-            "-XDcompilePolicy=simple"),
+            "-XDcompilePolicy=simple",
+            "--should-stop=ifError=FLOW"),
```

Comment:
This flag is necessary to reproduce the issue. We started building with it as of 512a3bebad373b7e655fdb84ef578073b54a2bcd, while the documentation generator was already introduced in 0f1507088350b014ebcca26193eb73929378d63d. This may explain why originally we omitted the error guard from the code.